src/providers/export/azureCustomVision.ts

```diff
@@ -15,62 +25,201 @@ export interface IAzureCustomVisionOptions {
     domainId?: string;
 }
 
+export interface ITagList {
+    [index: string]: IAzureCustomVisionTag;
+}
+
 export enum NewOrExisting {
     New = "New Project",
     Existing = "Existing Project",
 }
 
-export class AzureCustomVisionProvider extends ExportProvider<IAzureCustomVisionOptions> {
-    constructor(project: IProject, options: IAzureCustomVisionOptions) {
+/**
+ * @name - Azure Custom Vision Provider
+ * @description - Exports a VoTT project into an Azure custom vision project
+ */
+export class AzureCustomVisionProvider extends ExportProvider<IAzureCustomVisionExportOptions> {
+    private customVisionService: AzureCustomVisionService;
+    private assetService: AssetService;
+
+    constructor(project: IProject, options: IAzureCustomVisionExportOptions) {
         super(project, options);
         Guard.null(options);
+
+        const cusomVisionServiceOptions: IAzureCustomVisionServiceOptions = {
+            apiKey: options.apiKey,
+            baseUrl: "https://southcentralus.api.cognitive.microsoft.com/customvision/v2.2/Training",
```

Comment:
We should probably let the user provides also the baseUrl.  Different Azure Regions / DC may have different url

Reply:
Agreed as this service transitions into public preview... right now it is only supported via this region base url.